src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: sidebar
+title: Distribute Manifest V2 and V3 extensions
+permalink: /documentation/publish/distribute-manifest-versions/
+topic: Publish
+tags: [add-ons, beginner, tutorial, webextensions]
+contributors: [Rob--W, rebloor]
```

Comment:
Why is Rob mentioned here?

Reply:
Ha, nevermind. I removed Rob here for now. We'll add the exhaustive list of contributors to the draft if needed.